### ISSUE TITLE
Adding old exam to alea

### ISSUE THREAD

**Issue by behroozbc** [slatex/sTeX-React#437]
I have an idea regarding old exams. We can add them to the ALEA system, allowing students to answer and peer-review them. Reviewing the answers with actual answer classes will help students understand how their responses will be graded in the exam. In my opinion, the old exams should be in a separate section on the site, similar to homework.

**Comment by kohlhase** [comment]
Yes, that has always been my intention, which is why I have collected all of them and published them for AI-1/2. But you are right, we are not doing enough with that. Now that we have peer grading we can. 

 I think that the exam questions should appear in multiple places. 
 * A top-level "practice with old exams" section would be an obviously useful thing. But maybe we want to have a more general exam preparation page like the one Jonas has proposed [here](https://courses.voll-ki.fau.de/exp/pp_students) (Dirk Böhme is thinking about these things for his Master's Thesis). 
 * They already appear in the "problems anywhere" locations (traffic light, review),  but we can a) not filter for them and b) they are not flagged as exam problems (they could have a badge like "AI-1 Exam 2023" which would make them attractive to students. 
 * ...

**Comment by kohlhase** [comment]
I will be happy to help design this here, but I will not be able to implement this.

**Comment by kohlhase** [comment]
In fact it would be great to have this feature before the next exam preparation time. In fact @behroozbc I suspect you might have a personal interest in getting this done.

**Comment by kohlhase** [comment]
this is done,.
